include/crpropa/module/EMDoublePairProduction.h

```diff
@@ -27,7 +27,7 @@ class EMDoublePairProduction: public Module {
 
 public:
 	EMDoublePairProduction(
-		PhotonField photonField = CMB, //!< target photon background
+		PhotonField photonField, 	   //!< target photon background
```

Comment:
As you change from a small type to a class, we need to change all constructors here to use ref_ptr, i.e. 

EMDoublePairProduction(ref_ptr<PhotonField> photonField)